### DISCUSSION TITLE
It doesn't work, when the width and height of the image is too large.

### DISCUSSION THREAD

**Discussion by kimmy-wang** [iddan/react-native-canvas#252] [Q&A]
1. It doesn't work, and the display is a white screen, when the width and height of the image is too large. The width is `4032`, and the height is `3024`.
2. It works when the width is `1668`, and the height is `2500`.

[repo](https://github.com/upcwangying/react-native-canvas-demo)

**Comment by denisugo** [comment]
Have the same issue, but for me it only works if I use images with width of something around 800
